packages/core/src/servient.ts

```diff
@@ -210,18 +213,36 @@ export default class Servient {
 
     // will return WoT object
     public async start(): Promise<typeof WoT> {
+        if (this.#startedWoT != null) {
+            debug("Servient started already -> nop -> returning previous WoT implementation");
+            return this.#startedWoT;
+        }
+        if (this.#shutdown) {
+            throw Error("Servient cannot be started (again) since it was already stopped");
+        }
+
         const serverStatus: Array<Promise<void>> = [];
         this.servers.forEach((server) => serverStatus.push(server.start(this)));
         this.clientFactories.forEach((clientFactory) => clientFactory.init());
 
         await Promise.all(serverStatus);
-        return new WoTImpl(this);
+        return (this.#startedWoT = new WoTImpl(this));
     }
 
     public async shutdown(): Promise<void> {
+        if (this.#startedWoT === undefined) {
+            throw Error("Servient cannot be shutdown, wasn't even started");
+        }
```

Comment:
Seems like the CoAP tests are failing because they are calling the Servient's `shutdown` method without invoking `start` first. Should we maybe just return here? That would solve the problem. Or is it better to throw an error here?

Reply:
@relu91 ?
According to https://github.com/eclipse-thingweb/node-wot/pull/1195#issuecomment-1880833020 Cristiano prefers to throw ...

Reply:
Why are they calling stop without starting it? For the reasons above I would not jus silently return here.

Reply:
resolved by https://github.com/eclipse-thingweb/node-wot/pull/1212